chapters/stress/index.md

```diff
@@ -0,0 +1,271 @@
+# Jina Cloud Benchmark & Deployment
+
+#### Purpose: Showcase how Jina scales and how you can deploy to AWS
+
+---
+
+#### Table of Contents
+
+- [Infrastructure](#infrastructure)
+- [Datasets](#datasets)
+  * [Image search](#image-search)
+  * [Text search](#text-search)
+- [Configuration](#configuration)
+  * [Sharding and scheduling parameters](#sharding-and-scheduling-parameters)
+  * [Functional/Indexers parameter](#functional-indexers-parameter)
+  * [Client](#client)
+- [Experiments](#experiments)
+  * [Experiment 1 - Redis](#experiment-1---redis)
+    + [Results](#results)
+  * [Experiment 2 - BinaryPbIndexer](#experiment-2---binarypbindexer)
+    + [Results](#results-1)
+- [Test it yourself](#test-it-yourself)
+
+## Infrastructure
+
+This example benchmark prioritizes a real-world scenario, where a user hosts Jina in the cloud. We use six machines in AWS, of instace size `c5.2xlarge` (8 vCPUs, 16GB RAM). We also add 100GB gp2 ssd. For the Vector Indexer machine we increased RAM to 32 GB.
+
+The instances are split as follows:
+
+- **Client**: from this machine we create the Flows and issue the client requests (indexing and querying)
+- **Flow**: this machine hosts the `Gateway` to the `Flow`
+- **Encoder**: this machine processes the raw data (text or image) with the respective `Encoders`
+- **Indexer**: this machine hosts the `Indexers`
+- **Ranker**: this machine hosts the `Redis` servers and the `Rankers`
+
+## Datasets
+
+To run these tests, we prepared two scenarios: text and image search. These are the most common use cases for neural search. 
+
+Note that in these tests, the quality of the results is not assessed, and therefore only the amount of returned matches is validated.
+
+### Image search
+
+The first scenario simulates an image search application. Images are randomly generated and indexed using a Flow with this topology:
+
+![Image Index Flow](image_index_flow.png)
+
+We use the `mobilenet_v2` `Encoder` with the `ImageTorchEncoder`.
+
+At query time, the Flow looks like this:
+
+![Image Query Flow](image_query_flow.png)
+
+In the image search scenario, there is no `Segmentation` and ranking. The results are returned according to the `distance` in embedding space returned by the `VectorIndexers`. 
+
+### Text search
+
+The text search topology is similar to the one above, but with a `Segmenter` (`Sentencizer`) and a `Ranker` (`SimpleAggregateRanker`). This means we are indexing on the chunk level of a `Document`. 
+
+The full indexing Flow can be seen below:
+
+![Text Index Flow](text_index_flow.png)
+
+The full search Flow can be seen below:
+
+![Text Query Flow](text_query_flow.png)
+
+## Configuration
+
+The configuration parameters of this experiment can be found in the `.env` file, whose values are set as environment variables and changed in the `flow` and `pod` yamls.
```

Comment:
`flows/` and `pods/` since this refers to the folders? Slashes make them more "folder-y"

Reply:
Not really. The folder is called `image` or `text`. I'll try to rephrase. Check again